libbeat/dashboards/import_dashboards.go

```diff
@@ -91,6 +95,8 @@ func DefineCommandLine() (*CommandLine, error) {
 	cl.flagSet.BoolVar(&cl.opt.OnlyDashboards, "only-dashboards", false, "Import only dashboards together with visualizations and searches. By default import both, dashboards and the index-pattern.")
 	cl.flagSet.BoolVar(&cl.opt.OnlyIndex, "only-index", false, "Import only the index-pattern. By default imports both, dashboards and the index pattern.")
 	cl.flagSet.BoolVar(&cl.opt.Snapshot, "snapshot", false, "Import dashboards from snapshot builds.")
+	cl.flagSet.StringVar(&cl.opt.Certificate, "ca", "", "Certificate for SSL client authentication.")
```

Comment:
@andrewkroh I was working in the meantime on a solution where you can pass the `-cert` the client certificate, `-certkey` certificate key, and `-ca` certificate authority. I will change the code to use the proposed names as `curl` does.